config.json

```diff
@@ -367,7 +367,11 @@
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
-
+        "bitwise-operations",
+        "control-flow-(if-else-statements)",
```

Comment:
We've switched to `snake_case` naming convention for topics to increase consistency among tracks, so could you please choose them from the following list - https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt

Please follow the convention

Reply:
Sorry for the delay, I had switched it to `snake_case` and removed topic that is not presented in the `TOPICS.txt`.